src/public/providers.ts

```diff
@@ -64,7 +69,7 @@ class ProviderKeyElement {
         headers: {
           'Content-Type': 'application/json'
         }
-      })
+      }).then(() => window.location.reload())
```

Comment:
instead of reloading the page, should we have a `render` function so that we are consistent with implementation across different pages? For example, the `render` function in this PR: https://github.com/garageScript/myProxy/pull/91/files